client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -181,8 +183,17 @@ class CalypsoifyIframe extends Component<
 		}
 	}
 
+	componentDidUpdate( prevProps: ConnectedProps ) {
+		const mediaGutenframeActions = this.props.mediaGutenframeActions;
```

Comment:
Could be simplified to:

```suggestion
		const { mediaGutenframeActions } = this.props;
```